_layouts/default.html

```diff
@@ -15,8 +15,8 @@
     crossorigin="anonymous"></script>
   <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5943a8719f8890cf"></script>
   <script src="//www.eucookie.eu/public/gdpr-cookie-consent.js" type="text/javascript"></script>
-  <script type="text/javascript" src="js/question.js"></script>
-  <script type="text/plain" src="js/question_with_cookie.js" data-cookie-if="preferences"></script>
+  <script type="text/javascript" src="/js/question.js"></script>
+  <script type="text/plain" src="/js/question_with_cookie.js" data-cookie-if="preferences"></script>
```

Comment:
This to references could be remove because this scripts are not used anymore, actually the two js files could be also removed from the repository. Can you please do this?

Reply:
Yes I will do that

Reply:
@herflis done

Reply:
@iam-abbas can you please delete the two .js files from the repository as well? :)

Reply:
Yeah sure

Reply:
Done